book/src/tech/dev/formats/scenarios.md

```diff
@@ -79,6 +79,14 @@ To import this JSON file into A/B Street:
 cargo run --bin import_traffic -- --map=data/system/us/seattle/maps/montlake.bin --input=/path/to/input.json
 ```
 
+Then run the game as follows:
+
+```
+cargo run --bin game -- --dev data/system/us/seattle/maps/montlake.bin
+```
+
+You will need to select the senario with the user interface.
```

Comment:
Or... `data/system/us/seattle/scenarios/montlake/minimal.bin`. Both work, of course.